plug-ins/groups/class_paladin.cpp

```diff
@@ -96,10 +118,13 @@ VOID_SPELL(Banishment)::run( Character *ch, Character *victim, int sn, int level
            && victim->getRace( ) != race_demon
            && victim->getRace( ) != race_golem))
     {
-        act_p("$C1 - вовсе не нечисть и не демон.", ch, 0, victim, TO_CHAR, POS_RESTING);
+        act_p("К сожалению, $C1 -- не нечисть, не демон и не богомерзкий голем.", ch, 0, victim, TO_CHAR, POS_RESTING);
         return;
     }
 
+    if ( is_safe( ch, victim ) )
```

Comment:
@fiorine `is_safe` check seems redundant here. For offensive spells, safety is already checked. For onFight progs, victim is already fighting with us, thus not safe.

Reply:
@ruffinakoza Fixed.